### ISSUE TITLE
Amount of the transfer is not fitting into the transfer window

### ISSUE THREAD

**Issue by djsatok** [near/rainbow-bridge-frontend#78]
Only 15 characters can fit into the amount field. This would for sure be not enough for the majority transfers given `decimals == 18` is a de-facto standard.

<img width="509" alt="Screenshot 2021-02-03 at 10 06 06" src="https://user-images.githubusercontent.com/5417224/106717035-ce7c7800-6607-11eb-9e2f-c274e5831ee3.png">

**Comment by corwinharrell** [comment]
This needs a js solution. We should define the `font-size` with a kind of `max` value, and then allow the font-size to decrease when the string exceeds the length of its parent container. In this case, the input.

We see similar solutions being used more and more in financial applications in order to account for this exact problem. Here is an example from Square Cash:

![dynamic font-size](https://user-images.githubusercontent.com/5934188/106756558-60cb4e80-65fd-11eb-9828-b6d107015a31.gif)

**Comment by djsatok** [comment]
This looks good, IMO. Also, a user can make a mistake during the process of inserting zeros. Should we somehow help him, using `decimals` of the ERC-20?

**Comment by corwinharrell** [comment]
@djsatok I don't think we should worry about putting 18 decimals in the input as placeholder. We should allow for decimal numbers as input though.

**Comment by djsatok** [comment]
@corwinharrell I agree.
For sure, the transfer amount at the moment is confusing people. Having a token ticker implies that a transfer of '1 DAI' is actually a transfer of '10^18 atto-DAI', while in fact in the frontend an input of '1 DAI' is treated as '1 atto-DAI'. Moreover, when people are entering '0.5 DAI', they just get an uncaught error. This problem should be addressed.

@chadoh, we need to have something like this (this is with `ethers.js`):

``` typescript
const decimals = await erc20.decimals();
const balance = await erc20.balanceOf(userAddress);
const transferAmount = (new ethers.BigNumber("1" + "0".times(decimals))).mul(new ethers.BigNumber.from(parsefloat(input)));
if (transferAmount.gt(balance)) throw "Not enough funds";
erc20.transferFrom(userAddress, connectorAddress, transferAmount)
```

**Comment by djsatok** [comment]
Fixed in #99
